scripts/gdprdeleteuser.php

```diff
@@ -0,0 +1,114 @@
+#!/usr/local/bin/php -q
+<?php
+/**
+ * DO NOT RUN THIS SCRIPT WILLY NILLY
+ * This script will completely yoink a users personal details, show credits, podcast credits and a bunch of other stuff.
+ * Only run this script if a user has explicity declared they wish to be forgotten and understand the consequences.
+ * 
+ * run with "php gdprdeleteuser.php [userid]"
+ */
+
+use \MyRadio\Database;
+use \MyRadio\MyRadioException;
+use \MyRadio\MyRadioError;
+use \MyRadio\MyRadioEmail;
+use \MyRadio\MyRadio\CoreUtils;
+use \MyRadio\ServiceAPI\MyRadio_User;
+
+ini_set('log_errors', 1);
+ini_set('error_log', '/tmp/php-mailparser-error.log');
+ini_set('display_errors', 'On');
+
+require_once '../src/Controllers/root_cli.php';
+
+$db = Database::getInstance();
+
+$time = strtotime("-1 year", time());
+$date = date("Y-m-d", $time);
+
+$userid = $argv[1];
```

Comment:
Given this is the super scary delete script, I'd add a "are you really really sure" confirmation section. Just to help prevent accidents

Reply:
changes made